commands/__tests__/auth.test.ts

```diff
@@ -0,0 +1,57 @@
+// @ts-nocheck
+import yargs from 'yargs';
+import { addConfigOptions, addTestingOptions } from '../../lib/commonOpts';
+
+jest.mock('yargs');
+jest.mock('../../lib/commonOpts');
+
+// Import this last so mocks apply
+import authCommand from '../auth';
+
+describe('commands/auth', () => {
+  describe('command', () => {
+    it('should have the correct command structure', () => {
+      expect(authCommand.command).toEqual('auth [type] [--account]');
+    });
+  });
+
+  describe('describe', () => {
+    it('should provide a description', () => {
+      expect(authCommand.describe).toBeDefined();
```

Comment:
I saw some of the existing tests are checking for exact values here. I'm actually leaning towards removing those tests in favor of just checking for the existence of a description. Changing a description isn't a breaking change. The two impactful variations of the description are whether or not it has a `[beta] and if it's undefined.

Reply:
I think the benefits of checking the exact value are that they should not be changing frequently, and in the rare event they do change, we confirm it by updating the test copy as well.  It blocks accidental changes from happening.

Reply:
Ok I'm fine with keeping these in 👍. I think you're right that adding these for descriptions should be fairly harmless since they don't change often. I do want us to keep it in mind that there's a point where we start to get diminishing returns by increasing the granularity of our tests. Using tests to catch changes that would alter command functionality makes clear sense to me, but catching non-breaking copy changes feels to me like something that could also be handled at review stage. In the CLI, testing for exact copy output is essentially visual regression testing, which is useful in some areas but can get clunky pretty quickly if things change often.

Reply:
Yeah honestly... I guess I don't feel super strongly but I'm not sure I see a ton of value in checking for exact strings here. They shouldn't change but also the odds of them accidentally being changes (and passing a review) seem very low